New package: BQPD_jll v1.0.0+0

JuliaRegistrator opened the pull request:
- Registering package: BQPD_jll
- Repository: https://github.com/leyffer/BQPD_jll.jl
- Created by: @amontoison
- Version: v1.0.0+0
- Commit: 28ade38cfa0a567339b5feb33e8b5ed6d3524b51
- Reviewed by: @amontoison
- Reference: https://github.com/leyffer/BQPD_jll.jl/commit/28ade38cfa0a567339b5feb33e8b5ed6d3524b51#commitcomment-158469509
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6893c7d069b86ae542f5aeb3934dbc81523015b72814aa132db23f43a13906eee4c14a98a342192ba0c65d8cabf9c511878e4cd4e5fed07cf286e00ae8db8aecb288c8c98cc097c38a99b998bab9d14579c4aa306e8e2bc11bba992fcb5d5d2cff1b0eb754d3624647443c8889618dbfbbde90e055c694fa28fe8e51669e4b1306c1169460b3dd6ed27e1efa6e20edd2db34c786af8a313798c69b977f9daad9ab39026d46b100b41564b633925e88bb2 -->

amontoison commented:
[noblock] This PR is to register a "manual JLL" and Auto-registrator fails because of the _ in the name. (I have pinged on slack.)

The JLL are generated from closed source but we have permission to distribute the precompiled artifacts.
See the license in the tarballs.

odow commented:
[noblock]

From slack: I'd like some changes to the README first:

<img width="524" alt="image" src="https://github.com/user-attachments/assets/04124050-2870-4cc6-943f-2f4ba8b514d0" />

amontoison commented:
[noblock]

The pull request https://github.com/leyffer/BQPD_jll.jl/pull/1 aims to address the comments of @odow.

JuliaRegistrator committed:
New package: BQPD_jll v1.0.0+0

UUID: 1325ac01-0a49-589f-8355-43321054aaab
Repo: https://github.com/leyffer/BQPD_jll.jl.git
Tree: 23b35f6f5b4a2148ec9ee131502ee8143c3aa7e3

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

odow commented:
[noblock] @amontoison I think you can also remove the `+0` from the build number. If this PR is merged, then future updates will go through automatically. If we keep the build number, then I think they have to be manually merged every time (this happened to Gurobi_jll and Xpress_jll, so I removed the build number).

amontoison commented:
[noblock] Thanks @odow ! I was not aware of that. I removed the `+0` from the build number.

amontoison commented:
[noblock] A different PR #132004 was opened after removing the `+0`.
It should supersede this PR.
